### PR TITLE
Ap-5294: Turn Project Into An Express Server

### PR DESCRIPTION
**Pull request by codingwithcn**
## What Pull Request Does:
It modified the index.js file so that server can start receiving request. Only request currenly available is ```/health``` but it returns a static ```{ message: 'OK' }``` at least it should

## Major Changes:
Only changes I made was:

Modifiying the file: Install express server, and I edited the index.js file
Because I needed to turn project into a express server so I needed to download express first

## What To Test:
- No issue running the file
- The ```/health``` returns: ```{ message: 'OK' }```

## [Link To Jira Ticket](https://docnetwork.atlassian.net/browse/AP-5294)

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,4 +1,14 @@
...
+const express = require('express');
+const app = express();
 
+// Define routes
+app.get('/health', (req, res) => {
+  res.json({ message: 'OK' });
+});
+
+// Start the server
+app.listen(3000, () => {
```
Comment (AlienBob93) [inline index.js]: We should change the port to something other than 3000, since we use this port for the app as well. maybe 3010?

**Review by AlienBob93 — COMMENTED** [review]
Only have 1 bit of feedback, otherwise it looks good!

**Review by AlienBob93 — CHANGES_REQUESTED** [review]
I'll be good to approve once the port number has been changed.

**Comment by AlienBob93** [comment]
@codingwithcn there is a merge conflict.

File: index.js (hunk 1 above)
Comment (codingwithcn) [reply index.js]: recent commit should fix this

**Review by AlienBob93 — APPROVED** [review]
Looks good!

**Pull request merged**